steveluscher/wallet-adapter
# feature/build system

- first stab adding deps
- add emotion deps
- run preset-safe
- add mui and run preset-safe codemod
- remove butto=false to get MenuItem to work
- migrate theme
- fix MuiDialogTitle alignment
- switch to emotion styles
- use primary color for button text
- update walleticon
- update walletmultibutton
- fix wallet menu item styles
- clean up packages
- clean up package deps
- add mui to example/starter
- first stab at mui-v5 for starter/example
- remove legacy code
- update notify with styled
- remove unused mui/styles
- remote mui/styles
- remove unused dependencies
- organize imports / format
- fix dependencies
- wallets: add function to get all wallets
- starters: use getWallets
- use Record
- add details field
- react: patch version bump
- memo optimization
- organizing
- optimize ready check
- remove unused deps
- update yarn.lock
- reorganize to reduce churn
- wallets: patch version bump
- ENHANCE
- [chore] Write unit tests for WalletProvider
- [fix] Prevent a race condition when calling select() before the last adapter has disconnected
- [fix] The state initialization effect will no longer run if the wallet has changed
- use latest material-ui in wallet-adapter-example
- fix button text styling appearing uppercase
- fix menu position to be over anchor element
- use selected wallets for now
- [tidy] Remove state setters from dependency arrays
- Revert "[tidy] Remove state setters from dependency arrays"
- [enhancement] The setter in localStorage is now stable across renders
- Revert "Revert "[tidy] Remove state setters from dependency arrays""
- [tidy] select -> setName
- Fix up failing WalletProvider test
- [wallet readiness] Implement interface and readyState tracking
- [wallet readiness] Implement `readyState` in React `WalletProvider`
- [wallet readiness] Implement `readyState` in Vue
- [wallet readiness] Implement `readyState` in Svelte
- [wallet readiness] Implement `readyState` in Angular
- [wallet readiness] Show connected wallets first in the wallet modal
- space
- rename file
- torus no longer requires config
- use string enum
- refactor Wallet type out of base
- remove unused hook
- alias state value
- fix json format
- include public key in connect event
- refactor wallets into adapters, remove getters
- mark internal methods
- whitespace formatting
- refactor react package, WIP
- torus config optional
- organize package.json
- don't emit test files
- wallets -> adapters
- fix tests
- only build some packages for now
- fix react-based ui packages
- build react-based ui and starter packages
- minor version bump
- build(cjs/esm): add support for commonjs
- style(formatting): run prettier
